tests/gauge_alg_test.cpp

```diff
@@ -197,6 +198,7 @@ TEST_F(GaugeAlgTest, Generation)
   if (testgen) { ASSERT_TRUE(CheckDeterminant(detu)); }
 }
 
+#if defined(QUDA_TARGET_CUDA)
```

Comment:
why is here?

Reply:
Looks like I disabled the landau and coulomb tests for non-CUDA.  It may be ok to revert these now. To avoid casting I  initialize doubles with the .x .y and .z components (plaquette returns a double3). Reenabled with 21ea09d93